src/vm/eetwain.cpp

```diff
@@ -3085,6 +3085,17 @@ void EECodeManager::EnsureCallerContextIsValid( PREGDISPLAY  pRD, StackwalkCache
     _ASSERTE( pRD->IsCallerContextValid );
 }
 
+size_t EECodeManager::GetCallerSpUnsafe( PREGDISPLAY  pRD )
```

Comment:
I don't think it is worth extracting this simple expression into a function, especially when it is obvious what it does and it is used at two places only. Could you please revert this part of the change? Otherwise LGTM.